content/articles/postgresql-vs-mysql/index.md

```diff
@@ -0,0 +1,83 @@
+
+PostgreSQL, just like MySQL, is a relational data management system (RDMS). The two RDBMS store logical data separate from physical data. These are stored in tables, which are also called relations. The arrangement makes it possible for the user to manage physical data without causing any access trouble to data stored in the logical structures. A user, for example, can change the name of a stored file without altering its contents. Relational data management systems make it easier to handle multitudes of 'related' data.
+
+PostgreSQL was first released in 1996. MySQL made its initial release a year earlier. The existence of the databases for two decades and a half has garnered impressive popularity for them. Leading tech giants and business conglomerates employ the systems in their data management. Some organizations use the two systems within their entities and subsidiaries. However, most entities prefer one of the data management systems over the other for their reasons.
+
+### Features of PostgreSQL
+
+A good number of operating systems are compatible with PostgreSQL. These include [Windows OS](https://www.microsoft.com/en-us/windows), [Solaris](https://www.oracle.com/solaris/solaris11), [FreeBSD](https://www.freebsd.org/), [Linux](https://www.linux.org/), [UNIX](https://unix.org/), [Android](https://www.android.com/), and [OS X](https://www.android.com/). The RDBMS, written in [C](https://www.cprogramming.com/), boasts of programming interfaces for languages C, [C++](https://www.cprogramming.com/), [Python](https://www.python.org/), [Java](https://www.java.com/en/), [JavaScript](https://developer.mozilla.org/en-US/docs/Web/JavaScript), [Delphi](https://www.embarcadero.com/products/delphi), and [Perl](https://www.perl.org/). Other languages it supports include [Ruby](https://www.ruby-lang.org/en/), [Tcl](https://www.tcl.tk/), [Go](https://golang.org/), [Lisp](https://lisp-lang.org/), [Erlang](https://www.erlang.org/), [.NET](https://dotnet.microsoft.com/), and [Open Database Connectivity. ([ODBC](https://docs.microsoft.com/en-us/sql/odbc/reference/odbc-programmer-s-reference?view=sql-server-ver15)).
+
+Managing data for operations with heavy analysis is more accessible with PostgreSQL. It is an object-relational database, hence includes features like function overloading where one feature can do multiple tasks and inherit tables where properties of one table can be 'inherited' in another table. It also supports materialized views so that users can save a Structured Query Language (SQL) query on analysis on a disc as a physical table. This increases the order of magnitude speeds of complex SQL queries. As a result, the queries can interact more effectively with other SQL queries.
+
+The performance of PostgreSQL was also improved by its index design, which facilitates a comparatively quicker execution of CRUD (Create Read Update Delete) activities.
```

Comment:
Can you talk more about index design? A short overview of what it offers and why it was introduced?